core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -541,38 +567,33 @@ V remove(K key, V oldValue, Function<ByteBuffer, V> transformer) {
             throw new NullPointerException();
         }
 
-        boolean logical = true; // when logical is false, means we have marked the handle as deleted
-        Handle prev = null;
+        // when logicallyDeleted is true, it means we have marked the handle as deleted. Note that the entry may still be linked!
```

Comment:
Better to say, the entry will remain forever linked until rebalance happens. Remove doesn't disconnect entries from list

Reply:
fixed the comment